<!doctype html>
<html lang="en">

	<head>
		<meta charset="utf-8">

		<title>Summer research project</title>

		<meta name="description" content="Research project presentation to give a short talk at Bioinformatics Institute">
		<meta name="author" content="Peter Leontev">

		<meta name="apple-mobile-web-app-capable" content="yes" />
		<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />

		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">

		<link rel="stylesheet" href="css/reveal.css">
		<link rel="stylesheet" href="css/theme/simple.css" id="theme">

		<!--[if lt IE 9]>
		<script src="lib/js/html5shiv.js"></script>
		<![endif]-->
	</head>

	<style>
		figure {
			display: block; /* Блочный элемент */
			width: 480px; /* Ширина */
			height: 550px; /* Высота */
			float: left; /* Блоки выстраиваются по горизонтали */
			margin: 0 3px 3px 0; /* Отступы */
			text-align: center; /* Выравнивание по центру */
		}
		figure img {
			border: 2px solid #8b8e4b; /* Параметры рамки */
		}
		figure p {
			margin-bottom: 0; /* Отступ снизу */
		}
	</style>

	<body>
		<div class="reveal">

			<!-- Any section element inside of this container is displayed as a slide -->
			<div class="slides">
 				<section data-background="img/16S.png" data-background-size="390px" data-state="img-right">
					<h3>Metagenomic assembly and annotation of 16S ribosomal RNA genes by SPAdes</h3>
                    <p>
						<h7>Created by <a href="https:/ru.linkedin.com/pub/peter-leontev/63/348/67b">Peter Leontev</a></small><br>
                        Scientific advisors: <br>Sergey Nurk and Anton Korobeynikov, Center For Algorithmic Biotechnology SPbSU</h7>
					</p>
				</section>

                <section>
                    <h3>Why 16S rRNA genes are useful?</h3>
                    <p>16S rRNA genes are very important in reconstructing phylogenies due to slow evolution rates of their regions</p>
                    <figure>
                        <img height="400" data-src="img/woese_tree_of_life_16s.png"</img>
                        <figcaption><small>Woese et al. (1990)</small></figcaption>
                    </figure>
                    <figure>
                        <img height="400" data-src="img/tree_of_life_new.jpg"</img>
                        <figcaption><small>Christopher et al. (2015)</small></figcaption>
                    </figure>      
                </section>

                <section align="left">
                    <h3>16S rRNA genes sequencing</h3>
                    <ul>
                        <li><i>Target approach</i> allows to selectively capture genomic regions of interest from DNA sample;</li>
                        <li><i>Whole metagenome approach</i> provides sequencing information about the whole sample.</li>
                    </ul>   
                </section>

                <section>
					<h3>Direction I. Collaboration with JGI</h3>   
                    <ul id="fragments">
				        <br>
				        <li class="fragment">Source: Natalia Ivanova group from Joint Genomic Institute which studies very specific metagenomic datasets;</li>
				        <li class="fragment">Detection of unusual biology across different domains;</li>
                        <li class="fragment">Target sequencing lacks sensivity but there is a need to get full-length 16S rRNA genes;</li>
                        <li class="fragment">Additional challenge: necessity to get a genome information about the corresponding unusual group.</li>            
                    </ul>
				</section>

                <section>
                    <h3>Tasks</h3><br>   
                    <ul id="fragments"> 
                        <li class="fragment">Comparison of MetaSPAdes de novo and custom JGI pipeline assemblies.</li>
                        <li class="fragment">Contigs binning to 16S rRNA via SPAdes assembly graph;</li>                        
                    </ul>
                </section>

                <section>
                    <h3>Quantitative comparison of<br> assembled 16S rRNA sequences</h3><br>
                    <table>
                        <thead>
                            <tr>
                                <th align="center">Assembled by JGI custom pipeline</td>
                                <th align="center">Covered by MetaSPAdes contigs</td>
                            </tr>
                        </thead>
                        <tbody>                            
                            <tr>
                                <td align="center">1679</td>
                                <td align="center">1368*</td>
                            </tr>
                        </tbody>
                    </table>
                    <br>
                    <small>*Covered at least 80% of sequence. 1/5 missed 16S rRNA sequences have length less than 100 bp. </small>
                </section>

                <section>
                    <section>
                        <h3>Correspondence between 16S rRNA in the genome and in the assembly graph</h3>
                        <img data-src="img/correspondence.jpg" width="1000" height="500"</img>
                    </section>
                    <section>   
                        <h3>Bioinformatics reality</h3>
                        <img data-src="img/assembly_graph.png" height="545" width="1240"></img>
                        <small>https://github.com/rrwick/Bandage</small>
                    </section>
                </section>

                <section>
                    <h3>Detecting long contigs on isolates datasets</h3><br>
                    <table>
                        <thead>
                            <tr>
                                <th>Isolated organism</th>
                                <th>Long contigs* found</th>
                            </tr>
                        </thead>
                        <tbody>
                            <tr>
                                <td align="center">Some bacteria</td>
                                <td align="center">~80%</td>
                            </tr>
                        </tbody>
                    </table>
                    <br>
                    <small>*Aka corresponding information about an unusual group</small>
                </section>         

                <section> 
                    <section>
                        <h3>Direction II. Work in progress</h3>
                        <p>Assembly of full-length 16S rRNA genes via MetaSPAdes</p> 
                    </section>

                    <section>
                        <h3>Simple case of assembly</h3>
                        <p>When 16S rRNA is in a contig then one can easily <i>detect</i> it using the following tools:</p>
                        <ul id="fragments">
                            <li class="fragment">Barrnap (based on <i>NHMMER</i>) makes use of the primary structure conservation;</li>
                            <li class="fragment">Infernal (based on <i>Covariance Models</i>) takes into account both primary and secondary structures.</li>
                        </ul>
                    </section>

                    <section>
	                    <h3>Assembly of 16S rRNA genes on isolates</h3><br>
	                    <table>
	                        <thead>
	                            <tr>
	                                <th>16S rRNA copies in a Bacterial genome</th>
	                                <th>Assembled by SPAdes</th>
	                            </tr>
	                        </thead>
	                        <tbody>
	                            <tr>
	                                <td align="center">2-15</td>
	                                <td align="center">1+*</td>
	                            </tr>
	                        </tbody>
	                    </table>
	                    <br>    
	                    <small>
	                        *This number depends on available sequencing information (mate-pair, PacBio reads, etc.)   
	                    </small>            
                	</section>

                    <section>
                        <h3>Issue: local similarity of different 16S rRNA genes</h3>
                        <figure>
	                        <img height="500" data-src="img/conserved_regions.jpg"</img>
	                        <figcaption><small>Sequence diversity among the 16S rRNA genes in a genome was found in 235 species<br>
                            (from 0.06% to 20.38%; 0.55% ± 1.46%)</small></figcaption>
	                    </figure> 
                        <figure>
	                        <img height="500" data-src="img/msa_by_evol_distance.png"</img>
	                        <figcaption><small>Multiple alignments were built for 16S rRNA sequences for organisms from the same order(1-3), class (4-11), phylum (12)</small></figcaption>
	                    </figure> 	                    
	                </section>

                    <section>
                        <h3>Ideas about how to improve de novo assembly</h3>
                        <br>
                        <ul id="fragments">
                        	<li class="fragment">Use databases (Rfam, Greengenes) of known RNA families to add an information to the assembly graph.</li>
	                        <li class="fragment">Use information about both 16S rRNA primary and secondary structures (existing tool: Reago);</li>	                        
                        </ul>
                    </section>

                    <section>
                        <h3>Task</h3>
                        <p>Implement these ideas in MetaSPAdes.</p>
                    </section>

                </section>       

                <section>
                    <h2>THANK YOU! QUESTIONS?</h2>
                    <p><small><small> Hofstadter's Law: It always takes longer than you expect, even when you take into account Hofstadter's Law.</small></small></p>
                </section>
			</div>
		</div>

		<script src="lib/js/head.min.js"></script>
		<script src="js/reveal.js"></script>

		<script>

			// Full list of configuration options available at:
			// https://github.com/hakimel/reveal.js#configuration
			Reveal.initialize({
				controls: false,
				progress: true,
				history: true,
				center: true,
				slideNumber: true,
				hideAddressBar: true,

                

				transition: 'slide', // none/fade/slide/convex/concave/zoom

				// Optional reveal.js plugins
				dependencies: [
					{ src: 'lib/js/classList.js', condition: function() { return !document.body.classList; } },
					{ src: 'plugin/markdown/marked.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/markdown/markdown.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/highlight/highlight.js', async: true, condition: function() { return !!document.querySelector( 'pre code' ); }, callback: function() { hljs.initHighlightingOnLoad(); } },
					{ src: 'plugin/zoom-js/zoom.js', async: true },
					{ src: 'plugin/notes/notes.js', async: true }
				]
			});

            Reveal.configure({
                keyboard: {
                    39: 'next',
                    37: 'prev'
                }
            });

			window.addEventListener("mousedown", handleClick, false);
			window.addEventListener("contextmenu", function(e) { e.preventDefault(); }, false);

			function handleClick(e) {
				e.preventDefault();
				if(e.button === 0) Reveal.next(); 
				if(e.button === 2) Reveal.prev(); 
			}

		</script>

	</body>
</html>