<!doctype html>
<html lang="en">

	<head>
		<meta charset="utf-8">

		<title>Summer research project</title>

		<meta name="description" content="Research project presentation to give a short talk at Bioinformatics Institute">
		<meta name="author" content="Peter Leontev">

		<meta name="apple-mobile-web-app-capable" content="yes" />
		<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />

		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">

		<link rel="stylesheet" href="css/reveal.css">
		<link rel="stylesheet" href="css/theme/simple.css" id="theme">

		<!--[if lt IE 9]>
		<script src="lib/js/html5shiv.js"></script>
		<![endif]-->
	</head>

	<body>

		<div class="reveal">

			<!-- Any section element inside of this container is displayed as a slide -->
			<div class="slides">
 				<section data-background="img/16S.png" data-background-size="390px" data-state="img-right">
					<h3>Metagenomic assembly and annotation of 16S ribosomal RNA genes by SPAdes</h3>
                    <p>
						<h7>Created by <a href="https:/ru.linkedin.com/pub/peter-leontev/63/348/67b">Peter Leontev</a></small><br>
                        Scientific advisor: <br>Sergey Nurk, Center For Algorithmic Biotechnology SPbSU</h7>
					</p>
				</section>

                <section>
                    <section>
                        <h3>Why?</h3>
                        <p>16S rRNA genes are very important in molecular phylogeny due to slow evolution rates of their regions</p>
                    </section>
                    <section>
                        <h3>One great example [deprecated]</h3>
                        <a> 
                            <img height="450" data-src="img/woese_tree_of_life_16s.png"</img>
                        </a>
                        <p>Woese et al. (1990)</p>
                    </section>
                    <section>
                        <h3>Contemporary view</h3>
                        <a>
                            <img width="600" height="450" data-src="img/tree_of_life_new.jpg"</img>
                        </a>
                        <p>Christopher et al. (2015)</p>
                    </section>
                </section>

                <section align="left">
                    <h3>Analysis of 16S rRNA genes</h3>
                    <ul>
                        <li>Target sequencing;</li>
                        <li>Whole metagenomic sequencing.</li>
                    </ul>
                </section>

				<section id="fragments">
					<h3>The problem</h3>   
                    <ul>
				        <br><li class="fragment">We want to detect an unusual biology across Bacterial groups;</li>
                        <li class="fragment">We cannot use target assembly but we still need full length 16S rRNA genes;</li>
                        <li class="fragment">We also extremely need a genome information about an unusual group. What are we supposed to do?</li>                       
                    </ul>
                    <br><br><small class="fragment">Source: Joint Genomic Institute, Natalia Ivanova group</small>
				</section>

                <section id="fragments">
                    <h3>The plan</h3><br>   
                    <ul>
                        <li class="fragment">Contigs binning according to 16S rRNA via SPAdes assembly graph;</li>
                        <li class="fragment">Comparison of MetaSPAdes de novo and custom JGI pipeline assemblies.</li>
                    </ul>
                </section>

                <section>
                    <section>
                        <h3>Correspondence between 16S rRNA in the genome and in the assembly graph</h3>
                        <img data-src="img/correspondence.jpg" width="1000" height="500"</img>
                    </section>
                    <section>   
                        <h3>Bioinformatics reality</h3>
                        <img data-src="img/assembly_graph.png" height="545" width="1240"></img>
                        <small>https://github.com/rrwick/Bandage</small>
                    </section>
                </section>

                <section>
                    <h3>SPAdes 16S rRNA on isolates</h3>
                    <table>
                        <thead>
                            <tr>
                                <th>Organism</th>
                                <th>16S rRNA copies in genome</th>
                                <th>Assembled</th>
                                <th>Long contigs found/expected</th>
                            </tr>
                        </thead>
                        <tbody>
                            <tr>
                                <td align="center">Some bacteria</td>
                                <td align="center">2-15</td>
                                <td align="center">1+*</td>
                                <td align="center">~80%</td>
                            </tr>
                        </tbody>
                    </table><br>    
                    <small>
                        *This number depends on available sequencing information (mate-pair, PacBio reads, etc.)   
                    </small>            
                </section>

                <section>
                    <h3>Comparison of 16S rRNA from JGI pipeline and from MetaSPAdes</h3><br>
                    <table>
                        <thead>
                            <tr>
                                <th align="center">JGI</th>
                                <th align="center">SPAdes</th>
                            </tr>
                        </thead>
                        <tbody>
                            <tr>
                                <td align="center">1679</td>
                                <td align="center">1368</td>
                            </tr>
                        </tbody>
                    </table><br>
                    <small>*~1/5 missed 16S rRNA may be due to complex graph structures or simply misassemblies.</small>
                </section>

                <section> 
                    <section>
                        <h2>Future work</h2>
                    </section>
                    <section>
                        <h3>General 16S rRNA problem</h3>
                        <img width="1050" height="520" data-src="img/msa_by_evol_distance.png"</img>
                        <small>The same: 1-3 - order, 4-11 - class, 12 - phylum</small>
                    </section>
                    <section>
                        <h3>16S rRNA diversity</h3>
                        <img height="350" data-src="img/conserved_regions.jpg"</img>
                        <small>Sequence diversity among the 16S rRNA genes in a genome was found in 235 species (from 0.06% to 20.38%; 0.55% Â± 1.46%)</small>
                    </section>
                    <section>
                        <h3>How to improve de novo assembly?</h3>
                        <br><ul>
                            <li>Use information about 16S rRNA both primary and secondary structures (Reago);</li>
                            <li>Use existing DBs of RNA families (Rfam, Greengenes).</li>
                        </ul>
                    </section>
                    <section>
                        <h3>Long-term goal</h3>
                        Imrove MetaSPAdes de novo assembly using listed ideas.
                    </section>
                </section>

                <section>
                    <h2>THANK YOU!<h2>
                </section>
			</div>
		</div>

		<script src="lib/js/head.min.js"></script>
		<script src="js/reveal.js"></script>

		<script>

			// Full list of configuration options available at:
			// https://github.com/hakimel/reveal.js#configuration
			Reveal.initialize({
				controls: false,
				progress: true,
				history: true,
				center: true,
				slideNumber: true,
				hideAddressBar: true,

                

				transition: 'slide', // none/fade/slide/convex/concave/zoom

				// Optional reveal.js plugins
				dependencies: [
					{ src: 'lib/js/classList.js', condition: function() { return !document.body.classList; } },
					{ src: 'plugin/markdown/marked.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/markdown/markdown.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/highlight/highlight.js', async: true, condition: function() { return !!document.querySelector( 'pre code' ); }, callback: function() { hljs.initHighlightingOnLoad(); } },
					{ src: 'plugin/zoom-js/zoom.js', async: true },
					{ src: 'plugin/notes/notes.js', async: true }
				]
			});

            Reveal.configure({
                keyboard: {
                    39: 'next',
                    37: 'prev'
                }
            });

			window.addEventListener("mousedown", handleClick, false);
			window.addEventListener("contextmenu", function(e) { e.preventDefault(); }, false);

			function handleClick(e) {
				e.preventDefault();
				if(e.button === 0) Reveal.next(); 
				if(e.button === 2) Reveal.prev(); 
			}

		</script>

	</body>
</html>