<!doctype html>
<html lang="en">

	<head>
		<meta charset="utf-8">

		<title>Summer research project</title>

		<meta name="description" content="Research project presentation to give a short talk at Bioinformatics Institute">
		<meta name="author" content="Peter Leontev">

		<meta name="apple-mobile-web-app-capable" content="yes" />
		<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />

		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">

		<link rel="stylesheet" href="css/reveal.css">
		<link rel="stylesheet" href="css/theme/simple.css" id="theme">

		<!-- Code syntax highlighting -->
		<link rel="stylesheet" href="lib/css/zenburn.css">

		<!-- Printing and PDF exports -->
		<script>
			var link = document.createElement( 'link' );
			link.rel = 'stylesheet';
			link.type = 'text/css';
			link.href = window.location.search.match( /print-pdf/gi ) ? 'css/print/pdf.css' : 'css/print/paper.css';
			document.getElementsByTagName( 'head' )[0].appendChild( link );
		</script>

		<!--[if lt IE 9]>
		<script src="lib/js/html5shiv.js"></script>
		<![endif]-->
	</head>

	<body>

		<div class="reveal">

			<!-- Any section element inside of this container is displayed as a slide -->
			<div class="slides">
				<section data-background="#FFFFFF">
					<h3>Metagenomic assembly and annotation of 16S ribosomal RNA genes by SPAdes</h3>
                    <p>
						<h7>Created by <a href="https:/ru.linkedin.com/pub/peter-leontev/63/348/67b">Peter Leontev</a></small><br>
                        Scientific advisor: <br>Sergey Nurk, Algorithmic biology lab</h7>
					</p>
				</section>

				<section data-background="#FFFFFF">
					<h3>Purpose</h3>
					<p>
						Extract 16S rRNA genes and corresponding information from metagenomic assembly
					</p>
				</section>

				<!-- Example of nested vertical slides -->
				<section>
					<section>
						<h3>Why?</h3>
						<p>16S rRNA genes are very important in molecular phylogeny due to slow evolution rates of their regions</p>
					</section>
					<section>
						<h3>One great example [deprecated]</h3>
                        <a> 
                            <img height="450" data-src="img/woese_tree_of_life_16s.png"</img>
                        </a>
						<p>Woese et al. (1990)</p>
					</section>
					<section>
						<h3>Contemporary view</h3>
						<a>
							<img width="600" height="450" data-src="img/tree_of_life_new.jpg"</img>
						</a>
                        <p>Christopher et al. (2015)</p>
					</section>
				</section>

				<section>
                    <h3>Metagenomics in a nutshell</h3>
					<a>    
                        <img width="800" height="540" data-src="img/metagenomics_intro.png"</img>
                    </a>
                    <small>https://en.wikipedia.org/wiki/Metagenomics</small>                    
				</section>

				<section>
                    <section>
    					<h3>SPAdes 16S rRNA assembly for isolates</h3>
    					<table>
                            <thead>
                                <tr>
                                    <th>Organism</th>
                                    <th>16S rRNA copies in genome</th>
                                    <th>Assembled</th>
                                </tr>
                            </thead>
                            <tbody>
                                <tr>
                                    <td align="center">Some bacteria</td>
                                    <td align="center">2-15</td>
                                    <td align="center">1+*</td>
                                </tr>
                            </tbody>
                        </table><br>	
                        <small>
                            *This number depends on available sequencing information (mate-pair, PacBio reads, etc.)   
                        </small>			
                    </section>
                    <section id="fragments">
                        <h3>How to detect 16S rRNA?</h3><br>
                        <ul>
                            <li class="fragment">Easy: align known 16S rRNA to assembled contigs;</li>
                            <li class="fragment"><strong>Much better</strong>: use discriminative model (like HMM) including both 1° and 2° structures.</li>
                        </ul>
                    </section>
                </section>

				<section>
					<h3>Source of problems [exclude, probably]</h3>
					<img height="500" width="950" data-src="img/msa_by_evol_distance.png"</img>
                    <small>
                        The same <br>1-3: order, 4-11: class, 12: phylum
                    </small>
				</section>

				<section>
                    <section>
					    <h3>Correspondence between 16S rRNA in genome and in assembly graph</h3>
				        <img data-src="img/correspondence.png" width="1000" height="500"</img>
                    </section>
                    <section>
                        <h3>Bioinformatics reality</h3>
                        <img data-src="img/assembly_graph.png" height="550" width="1240"></img>
                    </section>
                </section>

				<section>
                    <section>
                        <h2>Results</h2>
                    </section>
                    <section>
					   <h3>Simulated dataset metagenomic assembly </h3><br>
    				    <table>
                            <thead>
                                <tr>
                                    <th align="center">Number of bacteria</th>
                                    <th align="center">16S rRNA expected/found</th>
                                    <th align="center">Long contigs expected/found</th>
                                </tr>
                            </thead>
                            <tbody>
                                <tr>
                                    <td align="center">45</td>
                                    <td align="center">45/39</td>
                                    <td align="center">320/118</td>
                                </tr>
                            </tbody>
                        </table>
				    </section>
                    <section>
                        <h3>Comparison of 16S rRNA from targeted assembly and from SPAdes contigs</h3><br>
                        <table>
                            <thead>
                                <tr>
                                    <th align="center">Expected number of 16S rRNA</th>
                                    <th align="center">Found in contigs</th>
                                </tr>
                            </thead>
                            <tbody>
                                <tr>
                                    <td align="center">1679</td>
                                    <td align="center">1368</td>
                                </tr>
                            </tbody>
                        </table><br>
                        <small>*311 missed 16S rRNA may be due to introns in some 16S or simply misassemblies.</small>
                    </section>
                </section>
                <section>
                    <h2>THANK YOU!<h3>
                </section>
			</div>
		</div>

		<script src="lib/js/head.min.js"></script>
		<script src="js/reveal.js"></script>

		<script>

			// Full list of configuration options available at:
			// https://github.com/hakimel/reveal.js#configuration
			Reveal.initialize({
				controls: true,
				progress: true,
				history: true,
				center: true,

				transition: 'slide', // none/fade/slide/convex/concave/zoom

				// Optional reveal.js plugins
				dependencies: [
					{ src: 'lib/js/classList.js', condition: function() { return !document.body.classList; } },
					{ src: 'plugin/markdown/marked.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/markdown/markdown.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/highlight/highlight.js', async: true, condition: function() { return !!document.querySelector( 'pre code' ); }, callback: function() { hljs.initHighlightingOnLoad(); } },
					{ src: 'plugin/zoom-js/zoom.js', async: true },
					{ src: 'plugin/notes/notes.js', async: true }
				]
			});

		</script>

	</body>
</html>
